docs/api/index.md

```diff
@@ -512,16 +511,205 @@ TODO
   ```
 
 ### toContain
+
+- **Type:** `(received: string) => Awaitable<void>`
+
+  `toContain` asserts if actual value is in an array. `toContain` can also check whether a string is a substring of another string
+
+  ```ts
+  import { expect, test } from 'vitest'
+  import { getAllFruits } from './stock'
+
+  test('the fruit list contains orange', () => {
+    expect(getAllFruits()).toContain('orange');
+  })
+  ```
+
 ### toContainEqual
 
+- **Type:** `(received: any) => Awaitable<void>`
+
+  `toContainEqual` asserts if an item with a specific structure and values is contained in an array.
```

Comment:
I think we should add it works like `toEqual` inside for each element

Reply:
okay.